services/scripts/testy-snapshot.sh

```diff
@@ -82,20 +98,27 @@ main() {
 
     # Validate database. Update the snapshot status on success/failure.
     aws ec2 create-tags --resources "$_snapshot_id" "$_volume_id" \
-                        --tags Key=Validation,Value=none
+                        --tags Key=Validation,Value=none Key=InstanceID,Value="$_instance_id"
 
     echo "Running validation script '$_validation_script' on volume '$_volume_id'."
-    var=$(date +%s%3N)
+    ts=$(date +%s%3N)
     if validate_database "$_validation_script" "$_mount_point" "$_snapshot_id" "$_volume_id"; then
         echo "Successfully validated database backup snapshot '$_snapshot_id'."
         aws logs put-log-events --log-group-name testy-logs \
                                 --log-stream-name testy-logs --log-events \
-            timestamp=$var,message="Backup snapshot validation succeeded. $_snapshot_id. $_instance_id"
```

Comment:
we use var instead of ts in a few more places - should we update those too?

Reply:
Sure, done in c17a8b75b38c653846ba048a0573221e0b8ad98e